package.json

```diff
@@ -2,7 +2,7 @@
   "name": "globalpayments-3ds",
   "version": "1.0.2",
   "description": "Helper library for leveraging 3DSecure 2 for Strong Customer Authentication (SCA)",
-  "main": "./dist/globalpayments-3ds.js",
+  "main": "./dist/globalpayments-3ds.esm.js",
```

Comment:
We should define this under `module` instead of `main`. Webpack, Rollup, etc should still be able to load the ES module at that point.

We will also want to support Commonjs (probably by switching from `iife` to `umd` for the existing files) in the future, but that work can be in another set of changes.